tests/lib/rules/indent.js

```diff
@@ -3424,6 +3424,133 @@ ruleTester.run("indent", rule, {
                 )
             `
         },
+        {
+            code: unIndent`
+                const foo = a.b(),
+                    longName =
+                        (memberFunction(
```

Comment:
I think this function name might be confusing since the function callee is just an identifier, not part of a MemberExpression (either with this as the object, or some other object). Could we change it just to like `func` or `doSomething` or something else?